routers/web/user/profile.go

```diff
@@ -256,6 +257,21 @@ func prepareUserProfileTabData(ctx *context.Context, showPrivate bool, profileDb
 				ctx.Data["ProfileReadme"] = profileContent
 			}
 		}
+	case "organizations":
+		orgs, count, err := db.FindAndCount[organization.Organization](ctx, organization.FindOrgOptions{
+			UserID:         ctx.ContextUser.ID,
+			IncludePrivate: showPrivate,
+			ListOptions: db.ListOptions{
+				Page:     page,
+				PageSize: pagingNum,
+			},
+		})
+		if err != nil {
+			ctx.ServerError("GetUserOrganizations", err)
+			return
+		}
+		ctx.Data["Cards"] = orgs
+		total = int(count)
```

Comment:
The below `pager.SetDefaultParams` and all `pager.AddParamString` could (should) be removed, and just use `AddParamFromRequest` (not really in this PR's scope, but it is good to have)

Reply:
5d657345aec57cbea438eea8bd62616c447395bf
I hope this is correct :)